### ISSUE TITLE
Reconsider forking frida-mono-api

### ISSUE THREAD

**Issue by Techbrunch** [NorthwaveNL/fridax#5]
I would be nice if `frida-mono-api` had it's own repo since the original repo does not appear to be maintained anymore.

**Comment by tijme** [comment]
I agree that the current situation isn't ideal. Forking would be better indeed. I'll look into it soon. Forking would also improve the UX of Fridax since it wouldn't require including the `frida-mono-api` by relative path anymore, allowing us to distribute Fridax as an NPM package.
